mappings/net/minecraft/world/biome/BasaltDeltaBiome.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5152 net/minecraft/world/biome/BasaltDeltaBiome
```

Comment:
```suggestion
CLASS net/minecraft/class_5152 net/minecraft/world/biome/BasaltDeltasBiome
```

Reply:
Is the biome officially called 'Basalt Deltas'?

Reply:
yes, its official name includes a `s`. Don't apply this suggestion but rename in enigma (the file needs a rename as well)

Reply:
Yep, it's the id